### PR TITLE
AJ-1030: allow multi-user WDS

### PR DESCRIPTION
**Pull request by davidangb**
## Summary of changes:
Removes a `Utils.cond` that prevented non-workspace-owners from seeing WDS data tables. As of this PR, all users can see WDS … *_if_* that WDS was deployed as multi-user.

**Reviewer:** the diff for this PR looks super ugly due to formatting/prettier changes. Please please ignore whitespace when viewing the PR.

### Testing strategy
Tested locally as two different users

### Visual Aids
As workspace owner:
Screenshot 5-5-2023 at 12 56 PM

As workspace reader BEFORE this PR:
Screenshot 2023-05-05 at 1 10 16 PM

As workspace reader AFTER this PR:
Screenshot 2023-05-05 at 12 57 05 PM

### CODE REVIEW

File: src/pages/workspaces/workspace/Data.js [hunk 1]
```diff
@@ -616,7 +616,7 @@ const WorkspaceData = _.flow(
       name,
       workspace,
       workspace: {
-        workspace: { createdBy, googleProject, attributes, workspaceId },
+        workspace: { googleProject, attributes, workspaceId },
```
Comment (davidangb) [inline src/pages/workspaces/workspace/Data.js:619]: `createdBy` is now unused

File: src/pages/workspaces/workspace/Data.js [hunk 2]
```diff
@@ -1384,163 +1384,149 @@ const WorkspaceData = _.flow(
...
-                            () => div({ style: { textAlign: 'center' } }, ['Select a data type from the navigation panel on the left'])
+                              icon('loadingSpinner'),
+                              ' Preparing your data tables, this may take a few minutes. ',
+                              Utils.cond(
+                                [
+                                  !uploadingWDSFile,
+                                  () =>
+                                    div({}, [
+                                      'You can ',
+                                      h(Link, { style: { marginTop: '0.5rem' }, onClick: () => setTroubleshootingWds(true) }, ['check the status']),
```
Comment (davidangb) [inline src/pages/workspaces/workspace/Data.js:1406]: formatting change only

File: src/pages/workspaces/workspace/Data.js [hunk 3]
```diff
@@ -1384,163 +1384,149 @@ const WorkspaceData = _.flow(
...
+                      extraMenuItems: h(
+                        Link,
+                        {
+                          href: `https://seqr.broadinstitute.org/workspace/${namespace}/${name}`,
+                          style: { padding: '0.5rem' },
+                        },
+                        [icon('pop-out'), ' Analyze in Seqr']
+                      ),
+                      noticeForPrefix: (prefix) =>
+                        prefix.startsWith(`${dataTableVersionsPathRoot}/`) ? 'Files in this folder are managed via data table versioning.' : null,
```
Comment (davidangb) [inline src/pages/workspaces/workspace/Data.js:1454]: formatting change only

File: src/pages/workspaces/workspace/Data.js [hunk 4]
```diff
@@ -1384,163 +1384,149 @@ const WorkspaceData = _.flow(
             div({ style: styles.tableViewPanel }, [
               _.includes(selectedData?.type, [workspaceDataTypes.entities, workspaceDataTypes.entitiesVersion]) &&
                 h(DataTableFeaturePreviewFeedbackBanner),
-              Utils.cond(
+              Utils.switchCase(
+                selectedData?.type,
                 [
-                  createdBy === getUser()?.email || isGoogleWorkspace,
+                  undefined,
```
Comment (aaronkanzer) [inline src/pages/workspaces/workspace/Data.js:1387]: Why `undefined` here? What would this represent?

**Review by yuliadub — APPROVED** [review]

**Comment by calypsomatic** [comment]
> Removes a `Utils.cond` that prevented non-workspace-owners from seeing WDS data tables. As of this PR, all users can see WDS … __if__ that WDS was deployed as multi-user.

If WDS was NOT deployed as multi-user what would happen?  Would the user just get a spinny wheel forever?

**Comment by davidangb** [comment]
> If WDS was NOT deployed as multi-user what would happen? Would the user just get a spinny wheel forever?

@calypsomatic great question - I just tested this, and yeah, it's an infinite spinner. I'll follow up with Beth to see if this is worth solving.

File: src/pages/workspaces/workspace/Data.js (hunk 4 above)
Comment (davidangb) [reply src/pages/workspaces/workspace/Data.js:1387]: I think you're seeing an artifact of the whitespace changes - the logic that uses `undefined` here hasn't changed.  Try ignoring whitespace when viewing this PR!

**Review by calypsomatic — APPROVED** [review]

**Pull request merged**